public_dropin_gpu_environments/nemollm_inference_ms/Dockerfile

```diff
@@ -12,8 +12,9 @@ RUN groupmod -g 1000 nemo && \
 # try and unpack any layers with out-of-bound UIDs
 FROM scratch
 COPY --from=0 / /
+ARG BASE_PATH=/opt/tritonserver/bin:/usr/local/cmake/bin:/usr/local/cmake/bin:/usr/local/mpi/bin:/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/ucx/bin:/opt/tritonserver/bin
 
-ENV PATH=/opt/tritonserver/bin:/usr/local/cmake/bin:/usr/local/cmake/bin:/usr/local/mpi/bin:/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/ucx/bin:/opt/tritonserver/bin \
+ENV PATH=${BASE_PATH} \
```

Comment:
Why this? e.g. why set this as an ARG? -- with the latest base image, hopefully this stuff all goes away because the UID is now 1000 for the nemo user.

Reply:
> with the latest base image

The latest image changes the required repository structure. Models compiled earlier for A10 are not working out-of-the box. I'll retry those latter.